sparklemotion/src/main/java/com/ifttt/sparklemotion/Decor.java

```diff
@@ -50,8 +50,8 @@
      */
     final boolean withLayer;
 
-    private Decor(@NonNull View contentView, @NonNull Page page, boolean layoutBehind, Animation slideInAnimation,
-            Animation slideOutAnimation, boolean withLayer) {
+    Decor(@NonNull View contentView, @NonNull Page page, boolean layoutBehind, Animation slideInAnimation,
```

Comment:
This change looks correct to me. This method is referenced by the inner class.
lolz, late to this comment.
would take a PR if you want, bast0079
not sure when a release will be cut, but thanks!